Resources/config/core.xml

```diff
@@ -66,7 +66,7 @@
         </service>
         <!-- twig -->
         <service id="sonata.admin.twig.global" class="Sonata\AdminBundle\Twig\GlobalVariables">
-            <argument type="service" id="service_container"/>
+            <argument type="service" id="sonata.admin.pool"/>
```

Comment:
In the case someone creates a service definition, using this service definition [as a parent](http://symfony.com/doc/current/service_container/parent_services.html). But maybe we could agree that probably no one does this @core23 ?